build.sc

```diff
@@ -1034,6 +1034,39 @@ def writeShortPackageVersionTo(dest: os.Path) = T.command {
   os.write.over(dest, version)
 }
 
+def importLauncher(directory: String = "artifacts") = T.command {
+  val nativeLauncher =
+    os.Path(cli.nativeImageDummy().path.toString.replace("nativeImageDummy", "nativeImage"))
+  val ext  = if (Properties.isWin) ".zip" else ".gz"
+  val from = os.Path(directory, os.pwd) / s"scala-cli-${Upload.platformSuffix}$ext"
+  System.err.println(s"Importing launcher from $from at $nativeLauncher")
+  if (!os.exists(from))
+    sys.error(s"$from not found")
+  val content: Array[Byte] =
+    if (Properties.isWin) {
+      import java.util.zip.ZipFile
+      Using.resource(new ZipFile(from.toIO)) { zf =>
+        val ent = zf.getEntry("scala-cli.exe")
+        Using.resource(zf.getInputStream(ent)) { is =>
+          is.readAllBytes()
+        }
+      }
+    }
+    else {
+      import java.io.ByteArrayOutputStream
+      import java.util.zip.GZIPOutputStream
+
+      val compressed = os.read.bytes(from)
+      val baos       = new ByteArrayOutputStream
+      Using.resource(new GZIPOutputStream(baos)) { gzos =>
+        gzos.write(compressed)
+      }
+      baos.toByteArray
+    }
+  os.write(nativeLauncher, content, createFolders = true)
```

Comment:
Instead of copying native luncher from artifacts, maybe we can unzip `scala-cli` native launchers from artifacts and then use it in integration test, just modify `TestUtil.cliPath` and read Scala CLI native launcher path from env path.

Reply:
I'll probably end up doing something like that, yeah. For now, the native-linux-tests job doesn't pick the imported launcher, and [keeps re-generating it](https://github.com/VirtusLab/scala-cli/runs/7523392761?check_suite_focus=true#step:7:2449).